client/src/assets/scss/argon-design-system/nav.scss

```diff
@@ -29,7 +30,7 @@
         transition: $transition-base;
 
         &:hover {
-        	color: $nav-pills-link-hover-color;
+            color: $nav-pills-link-hover-color;
         }
     }
```

Comment:
These should go into a new file in the custom folder so we are not overwriting the source of argon-design-system.

Reply:
done